settings/all_settings.go

```diff
@@ -17,5 +17,9 @@ func AllSettingsResources() map[string]common.Resource {
 	return map[string]common.Resource{
 		"default_namespace":         makeSettingResource[settings.DefaultNamespaceSetting, *databricks.WorkspaceClient](defaultNamespaceSetting),
 		"restrict_workspace_admins": makeSettingResource[settings.RestrictWorkspaceAdminsSetting, *databricks.WorkspaceClient](restrictWsAdminsSetting),
+		"compliance_security_profile_account": makeSettingResource[settings.CSPEnablementAccountSetting, *databricks.AccountClient](cspEnablementAccountSetting),
+		"enhanced_security_monitoring_account": makeSettingResource[settings.ESMEnablementAccountSetting, *databricks.AccountClient](esmEnablementAccountSetting),
+		"compliance_security_profile_workspace": makeSettingResource[settings.CSPEnablementSetting, *databricks.AccountClient](cspEnablementWorkspaceSetting),
+		"enhanced_security_monitoring_workspace": makeSettingResource[settings.ESMEnablementSetting, *databricks.AccountClient](esmEnablementWorkspaceSetting),
```

Comment:
@mgyucht We need to think about how to avoid this proliferation of resources... Can we have a single resource that will work on both workspace & account level...

Reply:
Sorry, took another look at this one. It seems like there are two separate settings, one about enabling these features at the workspace level, and one about enforcing them at the account level.